Update chain display names and NTT SDK packages

DBosley opened the pull request:
## Summary
- Fix UI to use chain display names instead of raw enum strings
- Update NTT SDK packages to version 3.0.3

## Changes
### UI Improvements
- Replace raw chain enum values with `displayName` property from chain config in user-facing text
- Affects "Time to" label in transaction details and gas slider prompts

### Dependency Updates
Update all @wormhole-foundation/*-ntt packages to version 3.0.3:
- sdk-definitions-ntt
- sdk-evm-ntt
- sdk-route-ntt
- sdk-solana-ntt
- sdk-sui-ntt

## Test plan
- [ ] Verify chain names display correctly in transaction details
- [ ] Verify chain names display correctly in gas slider UI
- [ ] Test with various chain combinations

DBosley committed:
fix: use chain display names instead of enum strings in UI

Replace raw chain enum values with displayName property from chain config
in user-facing text. Affects "Time to" label in transaction details and
gas slider prompts.

DBosley committed:
chore: update NTT SDK packages to 3.0.3

Update all @wormhole-foundation/*-ntt packages to version 3.0.3:
- sdk-definitions-ntt
- sdk-evm-ntt
- sdk-route-ntt
- sdk-solana-ntt
- sdk-sui-ntt

kev1n-peters commented on src/views/v3/Bridge/GasSlider/index.tsx:
```diff
@@ -158,7 +159,9 @@ const GasSlider = (props: {
       <Stack direction="row" alignItems="center" justifyContent="space-between">
-        <Typography>{`Need extra ${destGasToken.symbol} on ${destChain}?`}</Typography>
+        <Typography>{`Need extra ${destGasToken.symbol} on ${
+          destChainConfig?.displayName || destChain
```
Can we use `chainDisplayName()` instead for these cases?

DBosley committed:
chore: update OFAC SDN sanctioned wallets list

Automated update of SANCTIONED_WALLETS based on latest OFAC SDN list.

DBosley committed:
refactor: use chainDisplayName helper for display names

Replace manual config.chains[].displayName access with the existing
chainDisplayName() utility function for consistency.
